src/kudu/consensus/log_cache.cc

```diff
@@ -84,6 +84,14 @@ METRIC_DEFINE_gauge_int64(server, log_cache_msg_size, "Log Cache Message Size",
                           MetricUnit::kBytes,
                           "Size of the incoming uncompressed payload for the "
                           "messages in the log_cache");
+METRIC_DEFINE_gauge_int64(server, log_cache_compressed_payload_size,
+                         "Log Cache Compressed Payload Size",
+                          MetricUnit::kBytes,
+                          "Size of the compressed msg payload that is sent over the wire");
+METRIC_DEFINE_gauge_int64(server, log_cache_payload_size,
+                         "Log Cache Payload Size",
+                          MetricUnit::kBytes,
+                          "Size of the msg payload that is written to the log");
```

Comment:
Likewise, similar to the above, maybe would be better to describe this as "Total bytes of message payload written to the log since startup"